tensorboard/plugins/scalar/scalars_plugin_test.py

```diff
@@ -68,6 +70,51 @@ def set_up_with_runs(self, run_names):
     context = base_plugin.TBContext(logdir=self.logdir, multiplexer=multiplexer)
     self.plugin = scalars_plugin.ScalarsPlugin(context)
 
+  def set_up_db(self):
+    self.db_path = os.path.join(self.get_temp_dir(), 'db.db')
+    self.db_uri = 'sqlite:' + self.db_path
+    db_module, db_connection_provider = application.get_database_info(
+        self.db_uri)
+    context = base_plugin.TBContext(
+        db_module=db_module,
+        db_connection_provider=db_connection_provider,
+        db_uri=self.db_uri)
+    self.core_plugin = core_plugin.CorePlugin(context)
+    self.plugin = scalars_plugin.ScalarsPlugin(context)
+
+  def generate_run_to_db(self, experiment_name, run_name):
+    tf.reset_default_graph()
+    global_step = tf.get_variable(
+      tf.GraphKeys.GLOBAL_STEP,
+      shape=[],
+      dtype=tf.int64,
+      initializer=tf.ones_initializer,
+      trainable=False,
+      collections=[tf.GraphKeys.GLOBAL_VARIABLES, tf.GraphKeys.GLOBAL_STEP],
+      caching_device='/cpu:0',
+      use_resource=True)
+    train_op = global_step.assign_add(1, read_value=False)
+    placeholder = tf.placeholder(tf.float32, shape=[3])
+
+    db_writer = tf.contrib.summary.create_db_writer(
+        db_uri=self.db_path,
+        experiment_name=experiment_name,
+        run_name=run_name,
+        user_name='user')
+
+    scalar_ops = None
+    with db_writer.as_default(), tf.contrib.summary.always_record_summaries():
+      tf.contrib.summary.scalar(self._SCALAR_TAG, tf.reduce_mean(placeholder))
```

Comment:
Optional, but FWIW, given that you aren't actually using the summary value in this case, it's a lot simpler I think to just use some constant value like 1 or 42 rather than adding complexity to the test with reduce_mean and a placeholder.